src/coreComponents/managers/FieldSpecification/FieldSpecificationBase.cpp

```diff
@@ -96,7 +96,9 @@ FieldSpecificationBase::getCatalog()
 
 
 void FieldSpecificationBase::postProcessInput()
-{}
+{
+//  LvArray::tensorOps::l2Norm<3>( m_direction );
```

Comment:
We use `m_direction` as a full vector (direction + magnitude). Perhaps we should change the name. Objections? @klevzoff @francoishamon 

![image](https://user-images.githubusercontent.com/5116892/112083111-b1fab780-8b43-11eb-8b9c-c2dfeb581eba.png)